src/components/ERACommons.js

```diff
@@ -0,0 +1,203 @@
+import React, { useState, useEffect } from 'react';
+import { a, div, button, label, h, h2, h3, h4, span } from 'react-hyperscript-helpers';
+import { get, merge } from 'lodash';
+import { find, getOr, isNil } from 'lodash/fp';
+import * as qs from 'query-string';
+import { AuthenticateNIH, User, Collections } from '../libs/ajax';
+import { Config } from '../libs/config';
+import eraIcon from '../images/era-commons-logo.png';
+import './Animations.css';
+
+export default function ERACommons(props) {
```

Comment:
I don't have an issue with the file renaming here, but for reference, it really is `eRA Commons`: https://www.era.nih.gov/eracommons-timeline.htm